agate-rest/src/main/java/org/obiba/agate/web/rest/user/UserProfileResource.java

```diff
@@ -43,6 +43,4 @@
     if (user == null) NoSuchUserException.withName(username);
     return dtos.asDto(user, true);
   }
-
```

Comment:
In fact no, because this resource is user dependent. More appropriate would be /users